Makefile

```diff
@@ -602,6 +603,14 @@ docker-build-main-image: copy-binaries-to-image-dir central-db-image
 		image/rhel
 	@echo "Built main image for RHEL with tag: $(TAG), image flavor: $(ROX_IMAGE_FLAVOR)"
 	@echo "You may wish to:       export MAIN_IMAGE_TAG=$(TAG)"
+ifeq ($(strip $(kind_available)),)
+else
+	@echo "kind installed, loading images"
+	@echo "Loading image $(DEFAULT_IMAGE_REGISTRY)/main:$(TAG) into kind"
+	kind load docker-image $(DEFAULT_IMAGE_REGISTRY)/main:$(TAG)
```

Comment:
I'm not a kind user, just being curious here, why do we need both image refs (this line and l612) there?

Reply:
This is necessary to support short image references like `stackrox/main:...`.

Reply:
Yes, but, my question is: why do we need to support both refs?

Reply:
Simplicity in referencing it when executing `kubectl edit`. 
I've not used it recently, removed it for now.